Xamarin.Forms.Core/Markup/ElementExtensions.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Markup
+{
+	public static class ElementExtensions
+	{
+		public static TElement Effects<TElement>(this TElement element, params Effect[] effects) where TElement : Element
+		{
+			for (int i = 0; i < effects.Length; i++)
+				element.Effects.Add(effects[i]);
+			return element;
+		}
+
+		public static TFontElement FontSize<TFontElement>(this TFontElement fontElement, double size) where TFontElement : Element, IFontElement
```

Comment:
why do you expose FontSize, and none of the other properties ?

Reply:
The `Font` helper exposes all properties; this `FontSize` helper is a lighter method for the most common case.